Gemfile

```diff
@@ -62,6 +62,7 @@ gem "pg",                                                    :require => false
 gem "pg-dsn_parser",                    "~>0.1.0",           :require => false
 gem "psych",                            "~>3.1",             :require => false # This can be dropped once we drop ruby 2.5
 gem "query_relation",                   "~>0.1.0",           :require => false
+gem "rack-attack",                      "~>6.5.0"
```

Comment:
Should this be `:require => false`?  We don't need it, for example, generic workers.

Reply:
Yeah, that should be fine.

Combined with the `if defined?(Rails::Server)` and a followup `require 'rack-attack'`, I think that will cover our needs.

Reply:
Updated.